var/spack/repos/builtin/packages/r-circstats/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCircstats(RPackage):
+    """Circular Statistics, from "Topics in Circular Statistics" (2001)
+
+    Circular Statistics, from "Topics in Circular Statistics" (2001) S.
+    Rao Jammalamadaka and A. SenGupta, World Scientific."""
+
+    homepage = "https://www.example.com"
```

Comment:
```suggestion
    homepage = "https://cloud.r-project.org/package=CircStats"
```

Reply:
Oh my goodness... I can't believe I forgot that... So sorry!